[2025-08-04T20:39:43.618+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-08-04T20:39:43.635+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: account_generator.generator_account_data scheduled__2025-04-14T00:00:00+00:00 [queued]>
[2025-08-04T20:39:43.645+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: account_generator.generator_account_data scheduled__2025-04-14T00:00:00+00:00 [queued]>
[2025-08-04T20:39:43.646+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-08-04T20:39:43.685+0000] {taskinstance.py:2890} INFO - Executing <Task(KafkaProduceOperator): generator_account_data> on 2025-04-14 00:00:00+00:00
[2025-08-04T20:39:43.690+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=3185) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-08-04T20:39:43.692+0000] {standard_task_runner.py:72} INFO - Started process 3196 to run task
[2025-08-04T20:39:43.692+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'account_generator', 'generator_account_data', 'scheduled__2025-04-14T00:00:00+00:00', '--job-id', '115', '--raw', '--subdir', 'DAGS_FOLDER/account_generator.py', '--cfg-path', '/tmp/tmpd0p0mtqj']
[2025-08-04T20:39:43.695+0000] {standard_task_runner.py:105} INFO - Job 115: Subtask generator_account_data
[2025-08-04T20:39:43.745+0000] {task_command.py:467} INFO - Running <TaskInstance: account_generator.generator_account_data scheduled__2025-04-14T00:00:00+00:00 [running]> on host c4e4284cebf8
[2025-08-04T20:39:43.809+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='GGCODE' AIRFLOW_CTX_DAG_ID='account_generator' AIRFLOW_CTX_TASK_ID='generator_account_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-14T00:00:00+00:00'
[2025-08-04T20:39:43.810+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-08-04T20:39:43.818+0000] {kafka_operator.py:16} INFO - Producing 100 records to account_data at broker:29092
[2025-08-04T20:39:43.827+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-08-04T20:39:43.827+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=account_generator, task_id=generator_account_data, run_id=scheduled__2025-04-14T00:00:00+00:00, execution_date=20250414T000000, start_date=20250804T203943, end_date=20250804T203943
[2025-08-04T20:39:43.919+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-08-04T20:39:43.942+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-08-04T20:39:43.946+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
